addons/udes_delivery_control/models/picking_extra_fields.py

```diff
@@ -13,11 +13,32 @@ class StockPickingExtras(models.Model):
     picking_ids = fields.One2many(
         comodel_name="stock.picking", inverse_name="u_extras_id", string="Pickings"
     )
-    # Unloading Fields
+    loading_type = fields.Selection(
+        [("unloading", "Unloading"), ("backloading", "Backloading"), ("both", "Both")],
```

Comment:
Nit: `both` seems a bit wrong to use here for the code value (although in the context of the full list it makes perfect sense). Might be worth getting some opinions on this. I suppose we would want to avoid something long like `unloading_and_backloading`.

Reply:
Let's use `unload`, `backload` and `unload_backload`.

Reply:
done!